tests/python/pants_test/integration/BUILD

```diff
@@ -70,3 +70,14 @@ python_integration_tests(
   sources = ['test_prelude_integration.py'],
   uses_pants_run=True,
 )
+
+python_tests(
```

Comment:
This should be `python_integration_tests(`. Then remove the `tag` and remove `'src/python/pants/testutil:int-test'`. Also, you should be able to remove `'src/python/pants/engine/internals:tests'` due to dep inference.